ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -587,6 +587,22 @@ DFSS:
   - discharge_fan_run_command
   - discharge_fan_run_status
 
+DF2SS:
+  description: "Two discharge fan run command and status (start/stop). With dampers"
+  is_abstract: true
+  implements:
+  - OPERATIONAL
+  opt_uses:
+  - supply_air_damper_status_1
```

Comment:
these can be two separate types, following the convention of `RAIDC3X`; call them `DAIDC2X` and `RAIDC2X` and add isolation to the fields if they are for isolation.

Reply:
Modified